### PR TITLE
CI-92 assessment orgs by standard

### PR DESCRIPTION
**Pull request by scottcowan**

### CODE REVIEW

File: src/Sfa.Das.Sas.Tools.MetaDataCreationTool/Services/AssessmentOrgsXlsxService.cs [hunk 1]
```diff
@@ -38,9 +40,24 @@ public AssessmentOrganisationsDTO GetAssessmentOrganisationsData()
 
                 using (ExcelPackage package = new ExcelPackage(new FileInfo(filePath)))
                 {
-                    GetAssessmentOrganisations(package, assessmentOrgs);
+                    assessmentOrgs = GetAssessmentOrganisations(package).ToList();
```
Comment (sachpatel) [inline src/Sfa.Das.Sas.Tools.MetaDataCreationTool/Services/AssessmentOrgsXlsxService.cs]: Don't think you need to call ToList here.

Comment (scottcowan) [inline src/Sfa.Das.Sas.Tools.MetaDataCreationTool/Services/AssessmentOrgsXlsxService.cs]: There would be multiple enumerations (ie: multiple queries to ES from lines 46 and 55)

Comment (scottcowan) [inline src/Sfa.Das.Sas.Tools.MetaDataCreationTool/Services/AssessmentOrgsXlsxService.cs]: There would be multiple enumerations (ie: multiple queries to ES from lines 46 and 55)

**Comment by scottcowan** [comment]
the next [build](https://sfa-gov-uk.visualstudio.com/Digital%20Apprenticeship%20Service/_build/index?buildId=20807&_a=summary) passed, it was pointing at the wrong ES for the integration tests

**Pull request merged**